bubblewrap.c

```diff
@@ -2216,6 +2237,13 @@ main (int    argc,
       prctl (PR_SET_SECCOMP, SECCOMP_MODE_FILTER, &seccomp_prog) != 0)
     die_with_error ("prctl(PR_SET_SECCOMP)");
 
+  /* Optionally limit memory. This must occur after the fork and
+   * before the exec* call. */
+  if (opt_limit_mem) {
+    limit_mem.rlim_cur = limit_mem.rlim_max = opt_limit_mem * 1048576;
+    setrlimit(RLIMIT_DATA, &limit_mem);
```

Comment:
I'd like to go with the safer method first, denying this option in setuid mode. I don't know if there's a way to reliably check the limit for all supported platforms. That seems like something that's less stable and could be done in the future.

Checking the cap is a better way. I'll investigate.